src/panels/lovelace/hui-root.ts

```diff
@@ -399,8 +406,17 @@ class HUIRoot extends LitElement {
       huiView.columns = this.columns;
     }
 
-    if (changedProperties.has("hass") && huiView) {
-      huiView.hass = this.hass;
+    if (changedProperties.has("hass")) {
+      if (
```

Comment:
No need for this check, Lit will ignore an update if it's `===`. Alternatively, you can also just call `computeUrl` instead of `this._isRTL` in the turnery statement. Since that part is only calculated when in edit mode.